src/frontend/src/components/layout.js

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from 'react';
+
+import Header from './Header';
+import SideDrawer from './SideDrawer';
+import Backdrop from './Backdrop';
+import TextArea from './TextArea';
+import Logo from './shared/Logo';
+
+import logo from '../images/logo.svg';
+
+import data from '../../../../dummyData.json';
+
+import '../css/style.css';
+
+const items = [
+  { link: '#', text: 'Home' },
+  { link: '#', text: 'Participants' },
+  { link: '#', text: 'About' },
+];
+
+class Layout extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sideDrawerOpen: false,
+      scrolled: false,
+    };
+  }
+
+  componentDidMount() {
+    window.addEventListener('scroll', () => {
+      const isTop = window.scrollY > 320;
+      if (isTop) this.setState({ scrolled: true });
+      else this.setState({ scrolled: false });
+    });
+  }
+
+  drawerToggle = () => {
```

Comment:
This is failing linting because it's a proposal not in ecmascript yet.  You either need to add babel and deal with this transform, or (better) just use standard ES6 classes and `.bind()` in the constructor so you can rely on `this` here.

Reply:
> 
> 
> If this lands, I found a bug for someone to fix in the future :P
> Not entirely sure why but the hamburger menu button doesn't seem to show up properly on my machine.
> ![VirtualBox_Fedora_20_01_2020_18_34_35](https://user-images.githubusercontent.com/36822198/72763906-ce076f80-3bb3-11ea-83c1-f3bacd95fcc1.png)
> I can get to the side drawer only if I click on the exact pixel.

I did not finish the hamburger button, up for grabs.